src/NuGet.Clients/VsExtension/VsCredentialProviderImporter.cs

```diff
@@ -76,6 +76,11 @@ public class VsCredentialProviderImporter
             _initializer = initializer ?? Initialize;
         }
 
+        // The export from TeamExplorer uses a named contract, so we need to import this one separately.
+        [Import("VisualStudioAccountProvider", typeof(IVsCredentialProvider), AllowDefault = true)]
+        public IVsCredentialProvider VisualStudioAccountProvider { get; set; }
```

Comment:
Agreed, will adjust accordingly. 

I'm wondering: using the fully qualified type name of the VisualStudioAccountProvider of TeamExplorer as the contract name would drastically reduce the odds that conflicts happen.

In addition, when a third-party credential provider exported using the same contract name ends up in this ImportMany, how do I assert that the one from TeamExplorer got actually imported?